internal/common/db.go

```diff
@@ -66,6 +66,7 @@ type Cluster struct {
 
 	IgnoredClusterValidations string `gorm:"type:text"`
 	IgnoredHostValidations    string `gorm:"type:text"`
+	Uploaded                  bool   `json:"uploaded"`
```

Comment:
Maybe add a comment here about what this field is. "Uploaded" is fairly generic.

Reply:
Added a comment! Is the explanation ok?